uniffi_docs/src/lib.rs

```diff
@@ -144,3 +227,50 @@ pub fn extract_documentation(path: &Utf8Path) -> Result<Documentation> {
         impls,
     })
 }
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn it_parses_a_md_description() {
```

Comment:
In Rust people tend to name tests in form `test_some_test` (although it's not like a forced convention or something).

See: https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html
Or: https://github.com/rust-random/rand/blob/dbbc1bf3176138c867f3d84c0c4d288119a5a84e/rand_distr/src/gumbel.rs#L91-L160

Reply:
Not sure where i read the other nomenclature. I prefer the one you mentioned `test_some_test` . Fixed at 03418a9043dbcfb34498ff79da5502763e1b1bb0 .